modules/utils.py

```diff
@@ -18,22 +19,41 @@
 # Local imports
 from static.vuln_notify import vuln_found_notify
 
+# """configuring overal how the loggin is done"""
+# logging.basicConfig(
+#     filename=strftime("./logs/%Y%m%d_%H%M.log"),
+#     format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+# )
+
 # To remove HTTPS warnings
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+
 def generate_cache_buster(length: Optional[int] = 12) -> str:
-    """ Generate a random string used as a cache buster """
+    """Generate a random string used as a cache buster"""
 
     if not isinstance(length, int) or length <= 0:
         raise ValueError("[!] Lenght of cacheBuster be a positive integer")
-    return ''.join(random.choice(string.ascii_lowercase) for i in range(length))
+    return "".join(random.choice(string.ascii_lowercase) for i in range(length))
+
+
+def configure_logger(
+    module_name: str, handler: logging.Handler = logging.NullHandler()
+) -> logging.Logger:
+    """Provides a logger instance set to the module provided with a default handler"""
+
+    logger = logging.getLogger(module_name)
+    logger.addHandler(handler)
+    logger.propagate = False
+    return logger
+
```

Comment:
Temporary logger configuration to allow quick switch and debugging for developers